woocommerce/woocommerce-ios
# Blaze: Update mappers for target options and remove stubbed response

Part of #11622 


## Description

This PR updates the mappers for Blaze target options following the recent changes in the API response [peeHDf-2fw-p2#comment-1780]. The stubbed responses for the target endpoints were also removed since the endpoints have been deployed [peeHDf-2fw-p2#comment-1804]. 

An issue with the target device endpoint: it returns an "All" option. I added an update to the device picker view to remove the All section, and posted a question for the backend team [peeHDf-2fw-p2#comment-1808]. I'll update the view later if needed, depending on their response.

## Testing instructions

- Log in to a store with at least one existing product and is eligible for Blaze.
- Navigate to the Products tab and select any public product.
- On the product form, select Promote with Blaze.
- Notice that the campaign creation form is presented.
- Select the language field, note that the real data is now displayed.
- Navigate back and select the device field; the real data should be displayed on this screen, too. The All field is an option from the API, so picking it does not auto select all the other options for now.

## Screenshots

Language picker | Device picker 
--- | ---
<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/0ecc9ce1-6ccb-444b-96b2-f3c79481c96b" width=320 /> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/31937521-fb83-489b-848b-ee2fe86ed73e" width=320 />


---
